### ISSUE TITLE
Integration Tests failing on v1-10-test and v1-10-stable

### ISSUE THREAD

**Issue by kaxil** [apache/airflow#13854]
After https://github.com/apache/airflow/issues/13751 was fixed, the CI failed on integrations tests: https://github.com/apache/airflow/runs/1751628905

```
popenargs = (['kinit', '-kt', '/etc/airflow.keytab', 'airflow'],), kwargs = {}
retcode = 1, cmd = ['kinit', '-kt', '/etc/airflow.keytab', 'airflow']

    def check_call(*popenargs, **kwargs):
        """Run command with arguments.  Wait for command to complete.  If
        the exit code was zero then return, otherwise raise
        CalledProcessError.  The CalledProcessError object will have the
        return code in the returncode attribute.
    
        The arguments are the same as for the call function.  Example:
    
        check_call(["ls", "-l"])
        """
        retcode = call(*popenargs, **kwargs)
        if retcode:
            cmd = kwargs.get("args")
            if cmd is None:
                cmd = popenargs[0]
>           raise CalledProcessError(retcode, cmd)
E           subprocess.CalledProcessError: Command '['kinit', '-kt', '/etc/airflow.keytab', 'airflow']' returned non-zero exit status 1.
```

```
=========================== short test summary info ============================
ERROR tests/cli/test_cli.py::TestWorkerServeLogs::test_serve_logs_on_worker_start
ERROR tests/cli/test_cli.py::TestWorkerServeLogs::test_skip_serve_logs_on_worker_start
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_get_conn
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_get_lb_policy_invalid_policy
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_get_lb_policy_no_host_for_allow_list
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_get_lb_policy_with_args
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_get_lb_policy_with_no_args
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_record_exists_with_keyspace_from_cql
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_record_exists_with_keyspace_from_session
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_table_exists_with_keyspace_from_cql
ERROR tests/contrib/hooks/test_cassandra_hook.py::TestCassandraHook::test_table_exists_with_keyspace_from_session
ERROR tests/contrib/hooks/test_redis_hook.py::TestRedisHook::test_real_get_and_set
ERROR tests/contrib/hooks/test_redis_hook.py::TestRedisHook::test_real_ping
ERROR tests/contrib/operators/test_redis_publish_operator.py::TestRedisPublishOperator::test_execute_hello
ERROR tests/contrib/sensors/test_mongo_sensor.py::TestMongoSensor::test_poke
ERROR tests/contrib/sensors/test_redis_pub_sub_sensor.py::TestRedisPubSubSensor::test_poke_false
ERROR tests/contrib/sensors/test_redis_pub_sub_sensor.py::TestRedisPubSubSensor::test_poke_true
ERROR tests/contrib/sensors/test_redis_sensor.py::TestRedisSensor::test_poke
ERROR tests/executors/test_celery_executor.py::TestCeleryExecutor::test_celery_integration_0_amqp_guest_guest_rabbitmq_5672
ERROR tests/executors/test_celery_executor.py::TestCeleryExecutor::test_celery_integration_1_redis_redis_6379_0
ERROR tests/executors/test_celery_executor.py::TestCeleryExecutor::test_error_sending_task
================ 3964 skipped, 3 warnings, 21 errors in 59.67s =================
```

**Comment by kaxil** [comment]
Can you take a look please @potiuk / @mik-laj ?

**Comment by potiuk** [comment]
I will take a look over weekend.

**Comment by potiuk** [comment]
Should be fixed now @kaxil . That was caused by an updated kerberos authentication in 2.0 (we are using different image and authenticatio there - Kamil added it when he added Presto + Kerberos integration. We are still using the old image/confifguration in 1.10 but the intialisation of Kerberos configuration was removed from the CI scripts. 

I re-added it. 

I also noticed that the UndefinedJiinjaRule hanged - in a number of test jobs. People experienced similar hangs when they run the checks, so I removed it in thee next commit. I hope to get to green now (note I have not done anything with stable branch).
